src/rprocessing/RLangPApplet.java

```diff
@@ -214,8 +214,9 @@ protected void wrapProcessingVariables() {
         this.renjinEngine.put("displayHeight", displayHeight);
         this.renjinEngine.put("focused", focused);
         this.renjinEngine.put("keyPressed", keyPressed);
-        this.renjinEngine.put("frameCount", frameCount);
+        this.renjinEngine.put("frameCount", (double) frameCount);
```

Comment:
Yeah, actually I'm trying to find another way to solve the problem. For example do casts from float to double in the language level by renjin. It will reduce the manual work in BuiltinApplet.